sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/secret_client.hpp

```diff
@@ -132,6 +132,36 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Secrets {
         std::string const& name,
         GetSecretOptions const& options = GetSecretOptions(),
         Azure::Core::Context const& context = Azure::Core::Context()) const;
+
+    /**
+     * @brief Sets a secret in a specified key vault.
```

Comment:
I think we don't use present sentences. Change `Sets` to `Set`. Same for all other docs